site/en/docs/extensions/mv3/intro/mv3-overview/index.md

```diff
@@ -130,7 +130,8 @@ to an API method, this prevents the promise from being returned. Therefore you
 can defer this part of your migration until you're ready, or begin using
 promises immediately.
 
-Some scenarios, such as event listeners, will still require callbacks.
+Some scenarios, such as event listeners, will still require callbacks. For
+more information, see [Promises on MDN][3].
```

Comment:
MDN's article on promises is a great addition! 👍

I'm wondering if we should also add a link to the [Using promises](https://deploy-preview-3845--developer-chrome-com.netlify.app/docs/extensions/mv3/promises#compare-to-callback) article that explains how to convert from callbacks to promises 🤔. Just a thought

Reply:
Done.